### PR TITLE
Feature/pet page desktop

### LINKED ISSUE #156: Implement desktop version for Pet Page

### LINKED ISSUE #190: Improve styles for link to Starter pack on Pet page
Change color to blue
Change icon to Info icon
Remove space before question mark

### PR DESCRIPTION
**Pull request by okhabr**
Closes #190 
Closes #156 
- add desktop version of styles tor pet page ;
- correct styles for link to startter pack page ;

### CODE REVIEW

**Review by ivan-bonk — APPROVED** [review]
👍

**Review by Pashkaua — APPROVED** [review]
Cool !

File: src/pages/pet/components/observations/observations.module.scss [hunk 1]
```diff
@@ -17,5 +17,6 @@ $dark-grey: #8E8E93;
 }
 .plainText {
   font-size: 16px;
-  margin-bottom: 15px ;
+  margin-bottom: 15px;
+  margin-top: 6px;
 }
```
Comment (enheit) [inline src/pages/pet/components/observations/observations.module.scss:22]: nit: Don't forget to add empty line to the end of the file

File: src/pages/pet/pet.module.scss [hunk 2]
```diff
@@ -43,3 +44,44 @@
...
+    flex: 1;
+    .sectionHeader {
+      padding-top: 0;
+    }
+  }
+  .addContainer{
+    margin-top: 45px;
+    order: 2;
+  }
+}
```
Comment (enheit) [inline src/pages/pet/pet.module.scss:87]: nit: Don't forget to add empty line to the end of the file

**Review by enheit — APPROVED** [review]

File: src/pages/pet/components/observations/observations.module.scss (hunk 1 above)
Comment (okhabr) [reply src/pages/pet/components/observations/observations.module.scss:22]: +

File: src/pages/pet/pet.module.scss (hunk 2 above)
Comment (okhabr) [reply src/pages/pet/pet.module.scss:87]: +

**Pull request merged**